packages/app-mobile/ios/Joplin.xcodeproj/project.pbxproj

```diff
@@ -265,12 +265,12 @@
 				ORGANIZATIONNAME = joplinapp.org;
 				TargetAttributes = {
 					13B07F861A680F5B00A75B9A = {
-						DevelopmentTeam = A9BXAFS6CT;
```

Comment:
In fact I suspect all the changes in this file should be undone, because it looks like it's changes from your profile

Reply:
Thank you for catching this! I've reverted these changes.